encrypted-media/scripts/keystatuses.js

```diff
@@ -49,10 +49,11 @@ function runTest(config,qualifier)
             }
             function lexicographical( a, b ) { return a < b ? -1 : a === b ? 0 : +1; }
             function lexicographicalkey( a, b ) { return lexicographical( a.key, b.key ); }
-            var expected = [{ key: key1String, value: 'usable'}, { key: key2String, value: 'usable'}].sort( lexicographicalkey );
-            assert_equals(JSON.stringify(result),
-                          JSON.stringify(expected),
-                                 "keystatuses should have the two expected keys with keystatus 'usable'");
+            var expected1 = [{ key: key1String, value: 'usable'}, { key: key2String, value: 'usable'}].sort( lexicographicalkey );
+            var expected2 = [{ key: key1String, value: 'status-pending'}, { key: key2String, value: 'status-pending'}].sort( lexicographicalkey );
+            assert_in_array(    JSON.stringify(result),
+                                [ JSON.stringify(expected1),JSON.stringify(expected2) ],
+                                 "keystatuses should have the two expected keys with keystatus 'usable' or 'status-pending'");
```

Comment:
Not exactly. The specification constraint is that keystatuseschanged should be fired when one or more keystatuses do change, that the keyStatus values eventually become something other than "status-pending" and that they do not return to "status-pending" once they have moved to some other status.

So, the test (which is not the subject of the PR, btw), _could_ be more exacting in checking these constraints.